package.json

```diff
@@ -214,6 +214,23 @@
         "title": "%command.pull%",
         "category": "Hg"
       },
+      {
+        "command": "hg.purge",
+        "title": "%command.purge%",
+        "category": "Hg",
+        "icon": "$(discard)"
+      },
+      {
+        "command": "hg.purgeFiles",
+        "title": "%command.purgeFiles%",
+        "category": "Hg",
+        "icon": "$(discard)"
```

Comment:
`purge` is very different from discarding changes. If I saw this in the hg extension, I would assume it discards changes, and would be frustrated if it deleted my untracked files. This should not use the discard icon, because that icon implies reverting changes rather than deleting files. I don't think this command is used frequently enough to warrant a top-level icon in the SCM pane. It should just be a command in the command palette.

Reply:
The git extension uses the discard icon for untracked files, and it is really useful to be able to purge the files from the SCM view.
The tool also asks the user for confirmation, informing this is irreversible.

Also, the user has to enable purge extension for this to work, so they know what they are doing here.

Reply:
I guess the git extension uses the discard icon because it doesn't have a separate group for Untracked files, so it has to be consistent with the other files.

In our case we should use the trash icon instead. I believe this will be informing enough to the user that this action deletes files.

Icon reference: https://code.visualstudio.com/api/references/icons-in-labels